src/datasource.ts

```diff
@@ -557,18 +559,41 @@ export default class IrondbDatasource extends DataSourceApi<IrondbQueryInterface
     return this.irondbSimpleRequest('GET', queryUrl, false, true);
   }
 
-  metricTagValsQuery(metricQuery: string, cat: string, from?: number = null, to: number = null) {
-    let queryUrl = '/find' + this.getAccountId() + '/tag_vals?category=' + cat + '&query=';
-    queryUrl = queryUrl + metricQuery;
+  // Used by metricFindQuery
+  metricFindTagValsQuery(metricQuery: string, cat: string, from?: number = null, to: number = null) {
+    let queryUrl = '/find' + this.getAccountId() + '/tag_vals?category=' + cat + '&query=' + metricQuery;
```

Comment:
Is this supposed to be `/find123` or does `this.getAccountId()` return a leading slash? If not, this looks incorrect.

Reply:
getAccountId() {
    return this.irondbType === 'standalone' ? '/' + this.accountId : '';
  }